code/modules/mob/living/carbon/human/ai/brain/ai_brain.dm

```diff
@@ -92,9 +92,11 @@ GLOBAL_LIST_EMPTY(human_ai_brains)
 
 	if(!currently_busy && primary_weapon && current_target && !currently_firing && COOLDOWN_FINISHED(src, fire_overload_cooldown) && primary_weapon.has_ammunition())
 		currently_busy = TRUE
-		if(get_dist(tied_human, current_target) > gun_data.maximum_range)
+		var/target_futile = HAS_TRAIT(current_target, TRAIT_INCAPACITATED)
```

Comment:
is_mob_incapacitated() would be better to use than this

Reply:
done